operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -733,6 +742,21 @@ public String currentCaCertBase64() {
         return decoder.decode(caKeySecret().getData().get(CA_KEY));
     }
 
+    /**
+     * @return The base64 encoded thumbprint of the current CA certificate.
+     */
+    public String currentCaCertThumbprint() {
+        try {
+            X509Certificate cert = Ca.x509Certificate(currentCaCertBytes());
+            byte[] signature = MessageDigest.getInstance("SHA-256").digest(cert.getEncoded());
```

Comment:
Is there a reason to use SHA-256? Openssl and other tools use SHA-1. Maybe there's some value in our thumbprints being the same as theirs?